internal/wrappers/results-pdf-http.go

```diff
@@ -108,10 +109,11 @@ func (r *PdfHTTPWrapper) CheckPdfReportStatus(reportID string) (*PdfPollingRespo
 	}
 }
 
-func (r *PdfHTTPWrapper) DownloadPdfReport(reportID, targetFile string) error {
+func (r *PdfHTTPWrapper) DownloadPdfReport(url, targetFile string) error {
 	clientTimeout := uint(downloadTimeout)
-	customURL := fmt.Sprintf("%s/%s/download", r.path, reportID)
-	resp, err := SendHTTPRequest(http.MethodGet, customURL, http.NoBody, true, clientTimeout)
+	useAccessToken := FeatureFlags[MinioEnabled]
```

Comment:
leave a TODO for Elchanan here to change to the FF call

Reply:
He didn't change it .. he didn't have conflicts

Reply:
in Elchanan's pr